components/Chat/index.vue

```diff
@@ -133,862 +133,865 @@
+	showViewStats: false,
+	viewStats: {
+	  value: [ 0 ],
+	  current: 0,
+	  min: 0,
+	  max: 0,
+	  average: 0,
+	  total: 0,
+	},
       }
     },
 
     methods: {
       async connectToChat () {
 
-        await this.initChat();
+	await this.initChat();
 
-        const tokenUser = {
-          type  : 'unknown',
-          token : this.getChatToken,
-          page  : this.page,
-        };
+	const tokenUser = {
+	  type  : 'unknown',
+	  token : this.getChatToken,
+	  page  : this.page,
+	};
 
-        this.connectChat( tokenUser );
+	this.connectChat( tokenUser );
       },
 
       async onResize () {
-        await this.$nextTick( async () => await this.scrollToBottom( false ) );
+	await this.$nextTick( async () => await this.scrollToBottom( false ) );
       },
 
       async scrollToBottom ( force ) {
-        if ( this.$refs['chatmessages'] )
-          this.$refs['chatmessages'].scrollToBottom( force );
-        else
-          console.warn('Could not find scroll container for chat.');
+	if ( this.$refs['chatmessages'] )
+	  this.$refs['chatmessages'].scrollToBottom( force );
+	else
+	  console.warn('Could not find scroll container for chat.');
       },
 
       async addUserTag ( user ) {
-        this.$refs['chat-input'].$refs['chatmessageinput'].focus();
-        this.appendChatMessage( `@${user} ` );
+	this.$refs['chat-input'].$refs['chatmessageinput'].focus();
+	this.appendChatMessage( `@${user} ` );
       },
 
       async addWhisper ( user ) {
-        this.$refs['chat-input'].$refs['chatmessageinput'].focus();
-        this.setMessage( `/w @${user} ` );
+	this.$refs['chat-input'].$refs['chatmessageinput'].focus();
+	this.setMessage( `/w @${user} ` );
       },
 
       async authenticated ( user ) {
-        if ( this.unsubscribeUser ) this.unsubscribeUser();
-        if ( user ) {
-          await this.subscribeToUser( user.uid );
-        } else {
-          await this.logoutChat();
-
-          // Prevents edge case where troll users get connected twice
-          // This occurs during page loads for trolls only.
-          const tokenUpdated = !this.userToken || this.userToken.token !== this.getChatToken;
-          if ( tokenUpdated ) {
-            const tokenTroll = {
-              type: 'troll',
-              token: this.getChatToken,
-              page: this.page,
-            };
-            this.connectChat( tokenTroll );
-          }
-        }
+	if ( this.unsubscribeUser ) this.unsubscribeUser();
+	if ( user ) {
+	  await this.subscribeToUser( user.uid );
+	} else {
+	  await this.logoutChat();
+
+	  // Prevents edge case where troll users get connected twice
+	  // This occurs during page loads for trolls only.
+	  const tokenUpdated = !this.userToken || this.userToken.token !== this.getChatToken;
+	  if ( tokenUpdated ) {
+	    const tokenTroll = {
+	      type: 'troll',
+	      token: this.getChatToken,
+	      page: this.page,
+	    };
+	    this.connectChat( tokenTroll );
+	  }
+	}
       },
```

Comment:
The way this leaves a dangling } with each block makes me think a } is missing somewhere now that I keep seeing it